libraries/bot-connector/src/main/java/com/microsoft/bot/connector/authentication/ParameterizedBotFrameworkAuthentication.java

```diff
@@ -215,7 +215,7 @@ public BotFrameworkClient createBotFrameworkClient() {
         return jwtTokenValidationValidateAuthHeader(authHeader, activity.getChannelId(), activity.getServiceUrl());
     }
 
-    private CompletableFuture<ClaimsIdentity> jwtTokenValidationValidateAuthHeader(String authHeader,
+    CompletableFuture<ClaimsIdentity> jwtTokenValidationValidateAuthHeader(String authHeader,
```

Comment:
Almost forgot: Changing the modifier for the class to be able to mock it is not acceptable. Let's roll this back